GearRent/Networking/APIManager.m

```diff
@@ -0,0 +1,144 @@
+//
+//  APIManager.m
+//  GearRent
+//
+//  Created by Edwin Delgado on 7/20/22.
+//
+
+#import "APIManager.h"
+#import "Parse/Parse.h"
+#import "GNGeoHash.h"
+#import "CoreLocation/CoreLocation.h"
+#import "Edge.h"
+
+@implementation APIManager
+
++ (void)fetchListingsWithCoordinates:(NSArray *)coordinates completion:(void(^)(NSArray *listings, NSError *error))completion{
```

Comment:
Instead of using Objective C class methods to expose your utility functions, a better and cleaner way is to use C function so you only need to import the header with no need to call the class name. 

references 
- https://www.tothenew.com/blog/objective-c-utility-class-using-c-functions-in-ios/
- examples: https://fburl.com/code/q8i1i12m